airbyte-integrations/connectors/destination-astra/destination_astra/config.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.destinations.vector_db_based.config import VectorDBConfigModel
+from pydantic import BaseModel, Field
+
+
+class AstraIndexingModel(BaseModel):
+    astra_db_app_token: str = Field(
+        ...,
+        title="AstraDB Application Token",
+        airbyte_secret=True,
+        description="AstraDB Application Token",
+    )
+    astra_db_endpoint: str = Field(
+        ...,
+        title="AstraDB Endpoint",
+        airbyte_secret=True,
```

Comment:
Could we keep the endpoint non-secret? This is causing an issue in the UI when editing an existing configuration:
<img width="804" alt="Screenshot 2024-01-23 at 18 07 47" src="https://github.com/airbytehq/airbyte/assets/1508364/140d333a-5fd3-4c93-8fdc-eeeca0611976">

I think nobody ran into this problem yet. If something is marked as secret, two things happen:
* it's not shown in the UI in cleartext
* It is stored in a separate secret manager, not in the regular Airbyte DB with the rest of the configuration

I might miss something, but it seems like the endpoint isn't sensitive in this sense and can be stored with the rest of the configuration (the app token definitely is and should be declared a secret)

Reply:
Done